pom.xml

```diff
@@ -75,12 +89,12 @@
                         <configuration>
                             <rules>
                                 <rule>
-                                    <element>PACKAGE</element>
+                                    <element>BUNDLE</element>
```

Comment:
Why do you need BUNDLE here? I think it could pass with PACKAGE too if u have over 80% package coverage

Reply:
It's my agreement with Travis on github. Works locally without a bundle.